specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2024-09-01-preview/examples/RedisEnterpriseAccessPolicyAssignmentCreate.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "api-version": "2024-09-01-preview",
+    "subscriptionId": "e7b5a9d2-6b6a-4d2f-9143-20d9a10f5b8f",
+    "resourceGroupName": "rg1",
+    "clusterName": "cache1",
+    "databaseName": "default",
+    "accessPolicyAssignmentName": "defaultTestEntraApp1",
+    "parameters": {
+      "properties": {
+        "accessPolicyName": "default",
+        "user": {
+          "objectId": "6497c918-11ad-41e7-1b0f-7c518a87d0b0"
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "subscriptions/e7b5a9d2-6b6a-4d2f-9143-20d9a10f5b8f/resourceGroupsrg1/providers/Microsoft.Cache/redisEnterprise/cache1/databases/default/accessPolicyAssignments/default-TestEntraApp1",
+        "name": "defaultTestEntraApp1",
+        "type": "Microsoft.Cache/redisEnterprise/accessPolicyAssignments",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "accessPolicyName": "default",
+          "user": {
+            "objectId": "6497c918-11ad-41e7-1b0f-7c518a87d0b0"
+          }
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "id": "/subscriptions/e7b5a9d2-6b6a-4d2f-9143-20d9a10f5b8f/resourceGroupsrg1/providers/Microsoft.Cache/redisEnterprise/cache1/databases/default/accessPolicyAssignments/default-TestEntraApp1",
+        "name": "defaultTestEntraApp1",
+        "type": "Microsoft.Cache/redisEnterprise/accessPolicyAssignments",
+        "properties": {
+          "accessPolicyName": "default",
```

Comment:
Should this one (201) also have a provisioningState like the 200?

Reply:
yeah, updated.

Reply:
I made same behavior as it is in OSS